examples/python/prompt-chaining/agent.py

```diff
@@ -0,0 +1,55 @@
+from collections.abc import AsyncGenerator
+
+import beeai_framework
+from acp_sdk import Message
+from acp_sdk.client import Client
+from acp_sdk.models import MessagePart
+from acp_sdk.server import Context, Server
+from beeai_framework.backend.chat import ChatModel
+from beeai_framework.agents.react import ReActAgent
+from beeai_framework.memory import TokenMemory
+
+server = Server()
+
+async def run_agent(agent: str, input: str) -> list[Message]:
+    async with Client(base_url="http://localhost:8000") as client:
+        run = await client.run_sync(
+            agent=agent,
+            inputs=[Message(parts=[MessagePart(content=input, content_type="text/plain")])]
+        )
+
+    return run.outputs
+
+def get_text_from_messages(messages: list[Message]) -> str:
+    if not messages or not messages[0].parts:
+        return ""
+    return messages[0].parts[0].content
```

Comment:
no need to do it yourself, you can do `str(msg1 + msg2 + ..... + msg15)`

add and str operators are overriden